### PR TITLE
[alpha] Mention alpha build availability for testing

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by mit-mit** [comment]
@MaryaBelanger can you please review this?

File: src/null-safety/index.md
Comment (MaryaBelanger) [inline src/null-safety/index.md:123]: I'm wondering, if `dart --version` _doesn't_ report that version back, or I already know I _don't_ have the dart dev channel or flutter master channel set up, how can I fix that? Maybe the steps are described elsewhere on this page, I didn't check.

But just in case, can we anchor link on "the Dart dev channel"/"the Flutter master" channel to instructions for setting up those channels, even if it's just a couple lines somewhere? And/or where it says "make sure" `dart --version` reports <...>, point them to what they need to do to get it to report that.

File: src/null-safety/index.md
Comment (MaryaBelanger) [inline src/null-safety/index.md:125]: Kind of the same as the previous comment, "this _should_ ....": if it should do something, can we point them to some troubleshooting or something to do if it _doesn't_ do what it should? 

If the first comment I left about pointing them to how to set up those channels would ensure that these lines will behave as expected, then nothing else needs to be added here.

File: src/null-safety/index.md
Comment (mit-mit) [reply src/null-safety/index.md:123]: Good point. I'm fixing that by adding instructions on the download page:
https://github.com/dart-lang/site-www/pull/4548

File: src/null-safety/index.md
Comment (parlough) [inline src/null-safety/index.md:124]: If you're going to specify the `flutter` equivalent command for `analyze`, should probably do so here to:

```suggestion
$ dart pub get / flutter pub get     # this should resolve without issues
```

File: src/null-safety/index.md
Comment (parlough) [inline src/null-safety/index.md:128]: Maybe "migration status of your dependencies"?

File: src/null-safety/index.md
Comment (parlough) [inline src/null-safety/index.md:118]: This section got pretty large. Consider putting it in a subheader.

```suggestion
### Dart 3 alpha

As of late January 2023, **Dart 3 alpha** is available from the Dart dev channel
```

Comment (mit-mit) [reply src/null-safety/index.md:118]: I want this to be prominent, so keeping. It's only for some months anyway

**Comment by mit-mit** [comment]
@MaryaBelanger @parlough I'm going to merge this now, as it's needed for the blog post which is going out imminently.

Please fell free to tweak (even without waiting for my approval) in follow-up PRs.

**Review by parlough — APPROVED** [review]
Sounds good to me :)

**Pull request merged**
